Content.Server/GameObjects/Components/Markers/MobSpawnerComponent.cs

```diff
@@ -0,0 +1,29 @@
+using Robust.Shared.GameObjects;
+using Robust.Shared.IoC;
+using Robust.Shared.Random;
+using Robust.Shared.Serialization;
+using Robust.Shared.Serialization.Manager.Attributes;
+using Robust.Shared.ViewVariables;
+using System.Collections.Generic;
+
+namespace Content.Server.GameObjects.Components.Markers
+{
+    [RegisterComponent]
+    public class MobSpawnerComponent : Component, ISerializationHooks
```

Comment:
Don't need the hook as you're not calling the methods also file needs to be nullable